source/torso_calibration.rst

```diff
@@ -0,0 +1,91 @@
+Tutorial: Calibrating Fetch Torso
+=================================
+
+When the torso controller board is first powered, it will measurements two differnet
+position sensors to determine the absolute starting position of the torso.
+Once the absolute starting position of the torso is determined, the position measurement
+will retain millimeter precision as long torso remains powered.
+
+To work properly, the two torso sensors must be calibrated together.
+If the sensor are not properly calibrated, the calculation of the initial torso position
+could be incorrect.
+This should be easy to detect since the reported torso position will always be off by
+more than +/-10cm.  This problem is more likely to occur if the torso is powered on
+in the "up" position than the "down" position.
+
+After shipping the robot, it may be a good idea to verify torso calibration.
```

Comment:
we should be more specific about this -- the robot is factory calibrated, but if a user sees incorrect positions, they should verify the calibration